ProjectBasic/app/src/main/java/com/example/moviedb/screen/TabAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.moviedb.screen
+
+import android.content.Context
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import com.example.moviedb.screen.favorite.FavoriteFragment
+import com.example.moviedb.screen.home.HomeFragment
+
+class TabAdapter(fm: FragmentManager, context: Context) : FragmentStatePagerAdapter(fm) {
```

Comment:
update class name and remove context
```suggestion
class MainViewPageAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {
```